readme.md

```diff
@@ -48,16 +48,19 @@ It spits out to a JSON object, because that's the most portable system for using
 
 These people have helped make Name Your Contributors:
 
- - [@diasdavid](https://github.com/diasdavid)
+ - [@Berkmann18](https://github.com/Berkmann18)
+ - [@RichardLitt](https://github.com/RichardLitt)
```

Comment:
Just noticed that all but this line were in alphabetical order, I wonder why?

Reply:
Should we sort alphabetically, anyway?

Reply:
Really, we shouldn't have to rely on JQ for this.

Reply:
I don't think we should sort that way (other than by contributions) but that was what I noticed at first.
But I was playing around with `jq` and noticed it was the same kind of output but the README's one didn't seem to be up-to-date.

Reply:
Looks like we need a bot. :D

Reply:
That would be a good idea 😄.